### PR TITLE
Create darcoz.json

### PR DESCRIPTION
**Pull request by calvscart**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/darcoz.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "a website",
+    "repo": "https://github.com/DarcoZav/Darcoz.github.io",
+    "owner": {
+        "username": "DarcoZav",
+        "email": "zarconxorp@gmail.com",
+        "twitter": "ZarcoZxacy"
+    },
+    "record": {
+        "CNAME": "darcozav.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/darcoz.json:10]: 404?

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/darcoz.json (hunk 1 above)
Comment (DarcoZav) [reply domains/darcoz.json:10]: Sorry I forgot to remove the 404.html, and now it's fixed

Comment (WilliamDavidHarrison) [reply domains/darcoz.json:10]: @DarcoZav It still returns 404.

Comment (DarcoZav) [reply domains/darcoz.json:10]: Wait I make new repo

Comment (DarcoZav) [reply domains/darcoz.json:10]: I already remove the 404

Comment (WilliamDavidHarrison) [reply domains/darcoz.json:10]: @DarcoZav I still get a 404 when trying to reach https://darcozav.github.io

Comment (DarcoZav) [reply domains/darcoz.json:10]: Live site? 
https://darcozav.github.io/Darcoz.github.io/

Comment (WilliamDavidHarrison) [reply domains/darcoz.json:10]: @DarcoZav Rename the repo to `darcozav.github.io`

Comment (DarcoZav) [reply domains/darcoz.json:10]: @WilliamDavidHarrison done, i really sorry about that

**Review by wdhdev — APPROVED** [review]

**Pull request closed without merging**